bedrock/firefox/templates/firefox/campaign/unfck/index.html

```diff
@@ -72,27 +69,34 @@ <h3 class="c-item-title">{{ title }}</h3>
     </ul>
   {% endcall %}
 
-  <h2 class="c-section-title">The Unfcking Checklist</h2>
+  <h2 class="c-section-title">The Checklist</h2>
 
   <div class="mzp-l-content mzp-t-content-lg">
+    {% set tweet_text='2020 might be fcked, but the internet doesn’t have to be. Here are 5 easy ways we can all help turn things around.' %}
```

Comment:
Should this now be 6 easy ways?

Reply:
I think we're losing one of them and going back to 5. This is something I should double check prior to launch but I think it's okay to leave now.